docs/v0.4/invokers/java.md

```diff
@@ -48,14 +48,14 @@ public class UpperApplication {
 
 You can build your function either from local source or from source committed to a GitHub repository. For local build use:
 
-```bash
+```terminal
```

Comment:
`terminal` isn't a support language in [highlight.js](https://highlightjs.readthedocs.io/en/latest/css-classes-reference.html#language-names-and-aliases)

I've been using `sh`

Reply:
I'd prefer something that doesn't highlight any keywords, like sh or bash highlights `function`

Reply:
maybe just leave the language out although some editors complain - https://github.com/DavidAnson/markdownlint/blob/v0.16.0/doc/Rules.md#md040